src/logistration/RegistrationPage.jsx

```diff
@@ -138,9 +142,14 @@ class RegistrationPage extends React.Component {
   }
 
   render() {
+    if (this.isAccountCreationSuccessful()) {
```

Comment:
@adeelehsan has already done this in https://github.com/edx/frontend-app-logistration/pull/10.

Reply:
Okay would wait for his PR.

Reply:
Done